pmlc/conversion/gpu/gpu_to_vulkan.cc

```diff
@@ -181,7 +197,7 @@ class ConvertGpuLaunchFuncToVulkanCalls
   };
 
   llvm::SmallSet<mlir::Type, 4, mlirTypeComparator> bufferElementTypes;
-  llvm::SmallSet<const char *, 4> optionalSymbols;
+  llvm::SmallSet<StringRef, 4> optionalSymbols;
```

Comment:
As long as you're changing this -- maybe use `llvm::StringSet`?  (It's not completely clear to me what the ownership rules are, here.)

Reply:
Fixed